src/index.html

```diff
@@ -15,8 +15,44 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin ="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#logo">
+    <img src="images/logo.png" alt="Moyo logo" class="logo"/>
+      </a>
+
+    <nav class="nav">
+      <ul class="nav__list">
+        <li class="nav__item">
+      <a href="#apple" class="nav__link link1 is-active">APPLE</a>
+        </li>
+        <li class="nav__item">
+      <a href="#samsung" class="nav__link">SAMSUNG</a>
+        </li>
+        <li class="nav__item">
+      <a href="#smartphones" class="nav__link">SMARTPHONES</a>
+        </li>
+        <li class="nav__item">
+      <a href="#laptops_and_computers" data-qa="hover" class="nav__link link2 ">LAPTOPS & COMPUTERS</a>
```

Comment:
Don't use spaces in the 'a' tag's 'href' property. The 'href' for 'LAPTOPS & COMPUTERS' should be '#laptops_and_computers' without spaces.